### PR TITLE
Added title and course desc to getStudentData

### PR DESCRIPTION
**Pull request by zerfuth**
There might be an issue with duplicate courses being retrieved but not 100% sure

### CODE REVIEW

**Comment by si-x-li** [comment]
@zerfuth Are you using dev? There was some duplicate entries in course offerings

**Comment by zerfuth** [comment]
That could be the reason why, yeah.

File: backend/logic/users/users.js [hunk 1]
```diff
@@ -142,9 +142,11 @@ var getStudentData = async studentID => {
   let completedCourses, major, minors;
   try {
     completedCourses = await conn.query(
-      `SELECT course_code, semester
+    `SELECT course_offerings.course_code, course_offerings.semester,
```
Comment (KevenLYF) [inline backend/logic/users/users.js]: minor suggestion, consider using a variable to store a long query for better readability.

Comment (zerfuth) [reply backend/logic/users/users.js]: I attempted to indent it so it's still readable :')

**Pull request merged**